source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc

```diff
@@ -54,5 +54,7 @@ For details about IPv6 support, see link:{URL_virt_product_docs}{URL_format}admi
 |SSO with OpenID Connect |Adds support for external OpenID Connect authentication using Keycloak in both the user interface and with the REST API.
 |oVirt Engine Backup |Adds support to back up and restore {virt-product-fullname} {engine-name} with the Ansible `ovirt-engine-backup` role.
 |Failover vNIC profile |Allows users to migrate a virtual machine connected via SR-IOV with minimal downtime by using a failover network that is activated during migration.
+|Dedicated CPU policy |Guest vCPUs will be strictly pinned to a set of host pCPUs (similar to static CPU pinning). The set of pCPUs will be chosen to match the required guest CPU topology. If the host has an SMT architecture, thread siblings are preferred.
```

Comment:
better to call it `Dedicated CPU pinning policy` since this is how it is presented in the UI
and about `strictly pinned` - isn't it ambiguous? it is important to note here that the vcpus are exclusively pinned to the pcpus